webrtc.html

```diff
@@ -1010,7 +1114,7 @@
             connectivity being established.</p>
 
             <p>When the <dfn id=
-            "dom-peerconnection-updateice"><code>updateIce()</code></dfn>
+            "dom-peerconnection-setconfiguration"><code>setConfiguration()</code></dfn>
             method is invoked, the user MUST run the following steps to process
             the <code><a>RTCConfiguration</a></code> dictionary:</p>
```

Comment:
Where are the steps?

Also, I don't see anything about the peerIdentity stuff.  I'd need to think about it, but I think that the basic idea would be that it couldn't be unset once set, so removing or changing a pre-existing value wouldn't be allowed, even an absent value.  

Setting peerIdentity when there was previously none is tricky.  If there is an existing connection with a matching validated identity, that might be OK.  However, we can't permit one to be added, since that might allow the media isolation protections to be circumvented.